analyzers/tests/SonarAnalyzer.UnitTest/Rules/Hotspots/ExecutingSqlQueriesTest.cs

```diff
@@ -40,26 +40,26 @@ public class ExecutingSqlQueriesTest
         [TestCategory("Hotspot")]
         public void ExecutingSqlQueries_CS_Net46() =>
             Verifier.VerifyAnalyzer(@"TestCases\Hotspots\ExecutingSqlQueries_Net46.cs",
-                new CS.ExecutingSqlQueries(AnalyzerConfiguration.AlwaysEnabled),
-                GetReferencesNet46(Constants.NuGetLatestVersion));
+                                    new CS.ExecutingSqlQueries(AnalyzerConfiguration.AlwaysEnabled),
+                                    GetReferencesNet46(Constants.NuGetLatestVersion));
 
         [TestMethod]
         [TestCategory("Rule")]
         [TestCategory("Hotspot")]
         public void ExecutingSqlQueries_VB_Net46() =>
             Verifier.VerifyAnalyzer(@"TestCases\Hotspots\ExecutingSqlQueries_Net46.vb",
-                new VB.ExecutingSqlQueries(AnalyzerConfiguration.AlwaysEnabled),
-                ParseOptionsHelper.FromVisualBasic15,
-                GetReferencesNet46(Constants.NuGetLatestVersion));
+                                    new VB.ExecutingSqlQueries(AnalyzerConfiguration.AlwaysEnabled),
+                                    ParseOptionsHelper.FromVisualBasic15,
+                                    GetReferencesNet46(Constants.NuGetLatestVersion));
 
         internal static IEnumerable<MetadataReference> GetReferencesNet46(string sqlServerCeVersion) =>
-            Enumerable.Empty<MetadataReference>()
-                .Concat(FrameworkMetadataReference.SystemData)
-                .Concat(FrameworkMetadataReference.SystemDataOracleClient)
-                .Concat(NuGetMetadataReference.SystemDataSqlServerCe(sqlServerCeVersion)
-                .Concat(NuGetMetadataReference.MySqlData("8.0.22"))
-                .Concat(NuGetMetadataReference.MicrosoftDataSqliteCore())
-                .Concat(NuGetMetadataReference.SystemDataSQLiteCore()));
+            NetStandardMetadataReference.Netstandard
```

Comment:
for my education: why was it working before?

Reply:
In this test file we use `Verifier.VerifyAnalyzer` which adds the Netstandard if missing. See https://github.com/SonarSource/sonar-dotnet/blob/f75a387985c158f654dee052b644bf5b60c41b24/analyzers/tests/SonarAnalyzer.UnitTest/TestFramework/SolutionBuilder.cs#L77-L81

`Verifier.VerifyNoIssueReported` which is used by `SecurityHotspotTest` is not doing this so we have to add it manually. I think in the long term we will have to unify a bit the behavior.